libs/ui/src/lib/forms/Action/Action.style.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import { Button } from '../Button';
+
+import { CustomThemeProps } from '../../../typings/CustomThemeProps';
+
+type ActionProps = {
+  arrowPosition?: 'leading' | 'trailing';
+  wide?: boolean;
+  theme: CustomThemeProps;
+};
+
+export const Action = styled(Button)`
+  && {
+    display: flex;
+    justify-content: ${({ wide }: ActionProps) =>
+      wide ? 'space-between' : ''};
```

Comment:
Vou deixar essa sugestão aqui, porque dessa forma ele verifica se `wide` existe e então cria o `justify-content` ao invés de deixar ele com o valor vazio. 

```suggestion
    ${({ wide }: ActionProps) => wide && css`
      justify-content: 'space-between';
    `}
```

PS: Verifica se tá funcionando certinho, porque fiz no olho, então posso ter esquecido alguma coisa.